app/javascript/packs/application.tsx

```diff
@@ -462,6 +461,7 @@ import { highlightAll } from '../utils/highlight'
 import { AutomationLockedTooltipProps } from '../components/tooltips/AutomationLockedTooltip.js'
 import { ApproachesProps } from '@/components/track/Approaches'
 import { ChartData } from '@/components/impact/Chart'
+import { CreditsUsersProp } from '@/components/common/Credits.js'
```

Comment:
just noticing this `.js` at the end..

Reply:
I see a `.js` three lines above this line too